src/Api/CertificatesApi.php

```diff
@@ -0,0 +1,361 @@
+
+        return $response->json();
+    }
+
+    /* @see https://dm.realtimeregister.com/docs/api/ssl/products/get */
+    public function getProduct(string $product): Product
+    {
+        $response = $this->client->get('/v2/ssl/products/' . $product);
+
+        return Product::fromArray($response->json());
+    }
+
+    /* @see https://dm.realtimeregister.com/docs/api/ssl/products/list */
+    public function listProducts(
+        ?int $limit = null,
+        ?int $offset = null,
+        ?string $search = null,
+        ?array $parameters = null
+    ): ProductCollection {
+        $query = [];
+
+        if (! is_null($limit)) {
+            $query['limit'] = $limit;
+        }
+
+        if (! is_null($offset)) {
+            $query['offset'] = $offset;
+        }
+
+        if (! is_null($search)) {
+            $query['q'] = $search;
+        }
+
+        if (! is_null($parameters)) {
+            $query = array_merge($parameters, $query);
+        }
+
+        $response = $this->client->get('/v2/ssl/products', $query);
+
+        return ProductCollection::fromArray($response->json());
+    }
+
+    /* @see https://dm.realtimeregister.com/docs/api/ssl/request */
+    public function requestCertificate(
+        string $customer,
+        string $product,
+        int $period,
+        string $csr,
+        ?array $san = null,
+        ?string $organization = null,
+        ?string $department = null,
+        ?string $address = null,
+        ?string $postalCode = null,
+        ?string $city = null,
+        ?string $coc = null,
+        ?string $saEmail = null,
+        ?string $saLanguage = null,
+        ?array $approver = null,
+        ?array $dcv = null
+    ): void {
+        $payload = [
+            'customer' => $customer,
+            'product' => $product,
+            'period' => $period,
+            'csr' => $csr,
+        ];
+
+        if (! is_null($san)) {
+            $payload['san'] = $san;
+        }
+
+        if (! is_null($organization)) {
+            $payload['organization'] = $organization;
+        }
+
+        if (! is_null($department)) {
+            $payload['department'] = $department;
+        }
+
+        if (! is_null($address)) {
+            $payload['address'] = $address;
+        }
+
+        if (! is_null($postalCode)) {
+            $payload['postalCode'] = $postalCode;
+        }
+
+        if (! is_null($city)) {
+            $payload['city'] = $city;
+        }
+
+        if (! is_null($coc)) {
+            $payload['coc'] = $coc;
+        }
+
+        if (! is_null($saEmail)) {
+            $payload['saEmail'] = $saEmail;
+        }
+
+        if (! is_null($saLanguage)) {
+            $payload['saLanguage'] = $saLanguage;
+        }
+
+        if (! is_null($approver)) {
+            $payload['approver'] = $approver;
+        }
+
+        if (! is_null($dcv)) {
+            $payload['dcv'] = $dcv;
+        }
+
+        $this->client->post('/v2/ssl/certificates', $payload);
```

Comment:
Already discussed/talked about, but as a reminder the suggestion to return the created process ID. (Same for the renew request).